PresentScreenings/Entities/FilmInfo.cs

```diff
@@ -320,6 +342,44 @@ private static Film.FilmInfoStatus StringToFilmInfoStatus(string name)
             }
         }
 
+        /// <summary>
+        /// Search for HTML tags, remove them and apply a bold font
+        /// to the text range marked by the tags.
+        ///
+        /// It is presumed that:
+        ///     - Every start tag has a matching end tag
+        ///     - Tag ranges do not overlap
+        /// </summary>
+        /// <param name="attrText"> The attributed string to be searched from</param>
+        /// <param name="index">The index to start searching</param>
+        /// <returns>The new starting index, -1 when no tags are found.</returns>
```

Comment:
Please add a user story to make this more general, including different attributes for different tags.

Reply:
Done.